scenarios/25_The_Awakening.cfg

```diff
@@ -286,13 +287,15 @@
         canrecruit=yes
         side=2
         facing=ne
+        # Spearman = fighter, Javelineer = fighter, Bowman = archer, Peasant = fighter, Swordsman = fighter, Mage = mixed fighter
         recruit=Spearman,Javelineer,Bowman,Peasant,Swordsman,Mage
         {INCOME4 9 7 5 3}
         {GOLD4 250 220 190 160}
         [ai]
             {NO_SCOUTS}
             passive_leader=no
-            recruitment_pattern=fighter,fighter,healer,fighter
+            # FIXME: what to do about the "healer" entry here?
```

Comment:
@nemaara so, do you think we should:
1. Give Huon access to White Mages, to keep his faction entirely Loyalist
2. Give Huon access to Dune Herbalists, which, as Level 1 units, shouldn't disrupt the balance as much as White Mages might
3. Just remove the "healer" entry from this recruitment pattern
4. Just leave everything as is here

?

Reply:
White Mage is fine.

Reply:
OK, White Mage recruits allowed in [2c0ea94](https://github.com/nemaara/A_New_Order/pull/117/commits/2c0ea94256fc1d7a8a9d5c3bc00f751e2ab0500c)